src/nvim/ex_cmds.c

```diff
@@ -1635,14 +1635,14 @@ void print_line(linenr_T lnum, int use_number, int list)
 
   msg_start();
   silent_mode = FALSE;
```

Comment:
Missed a FALSE here

Reply:
Opted to NOLINT to avoid clint chain reaction.